internal/service/loadbalancer/loadbalancer.go

```diff
@@ -42,11 +45,16 @@ func ResourceLoadBalancer() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 			},
-			"network": {
-				Description: "Private network UUID where traffic will be routed. Must reside in load balancer zone.",
-				Type:        schema.TypeString,
-				ForceNew:    true,
-				Required:    true,
+			"networks": {
```

Comment:
I would use a singular form here, i.e. `network`. Singular form is a bit nicer from a user perspective, for me having to use a plural form to define a single network is a bit confusing. Not a blocker

Reply:
That is true. I was thinking to use singular form but examples that I viewed all used plural naming and `network` is still reserved to old field so I decided to go with plural.